src/engine/LoadingScreen.cs

```diff
@@ -25,17 +29,24 @@ public class LoadingScreen : Control
 
     private static LoadingScreen instance;
 
+    private readonly Random random = new Random();
+
     private Label artDescription;
     private Label loadingMessageLabel;
     private Label loadingDescriptionLabel;
     private Label tipLabel;
     private Control spinner;
 
-    private string loadingMessage = TranslationServer.Translate("LOADING");
+    private bool wasVisible;
+
+    private Timer randomizeTipTimer;
+
+    private string loadingMessage;
+    private string tip;
     private string loadingDescription = string.Empty;
-    private string tip = TranslationServer.Translate("LOADING_TIP");
```

Comment:
One more thing about this, doesn't your PR remove LOADING_TIP?
Could you maybe rescue that text into the microbe editor tips?